sass-style-guide.md

```diff
@@ -1,37 +1,21 @@
-## Where to place code?
+# Sass Styleguide
 
-### Order of rules within a Sass file
+## Inhaltsverzeichnis
 
-The order of style rules that should be applied depends on the type of stylesheet.
+- Einleitung
+- Manifest-Datei
+- Style-Definitionen
+- Syntax und Formatierung
+- Kommentare
+- Editorconfig
 
-#### Content-driven style definitions
+## Einleitung
 
-* The easiest way to read and find specific content-driven style rules is to visually follow the order of the content on the rendered page. Iterate in this order:
-  1. *Outside in*: Following ZASAF, you are likely to start with the page wrapper element.
-  2. *Left-to-right*: Next, you start with the left-most element, followed by elements in the same (grid) row.
-  3. *Top-to-bottom*: You go on top-to-bottom just how you would read that page
-* Some things are shared, like a variable definition that is referred in multiple parts of your stylesheet. Just put it on top of the first block that uses it. A situation that you should avoid is sharing variables across files. If you cannot avoid that, put it on top of the file.
-* Nestings within the namespace of an element start with pseudo classes, followed by pseudo elements, followed by modifiers, followed by direct children (`>`), followed by children further down the DOM tree (if even necessary). Example:
+Bei der Arbeit an großen, langlebigen Projekten mit mehreren Entwicklern unterschiedlicher Fähigkeiten ist es wichtig, dass wir alle einheitlich arbeiten. Unsere oberste Prämisse sollte sein, dass unsere Stylesheets auf der einen Seite leicht pflegbar und skalierbar sind und auf der anderen Seite unser Code aber transparent und lesbar bleibt. Um dies zu erfüllen, helfen folgende Empfehlungen.
```

Comment:
Wir nutzen in unserem Sprachgebrauch doch auch das englische Wort "Guidelines". Richtlinien wäre die beste Übersetzung, aber das ist weniger positiv konnotiert.